src/chatClient.h

```diff
@@ -223,7 +223,13 @@ class PeerChatRoom: public ChatRoom
     Contact *contact() const { return mContact; }
 
     /** @brief The screen name of the peer */
-    virtual const std::string& titleString() const;
+    virtual std::string titleString() const;
+
+    /** @brief Returns a string <fistname length><fistname><lastname>. It has binary layout
+      * First byte indicate first name length
+      */
+    virtual std::string completeTitleString() const;
```

Comment:
No need to create a new copy at every call. Please, keep a `const std::string &` as return datatype.